### PR TITLE
Enable quicklook settings

### PR DESCRIPTION
**Pull request by bjoernbroetz**
Adds config setting for quicklooks

### CODE REVIEW

**Comment by bjoernbroetz** [comment]
@axel-lauer @schlunma @hb326 @LisaBock : as discussed earlier this day.

File: esmvalcore/_config.py [hunk 1]
```diff
@@ -61,8 +62,13 @@ def read_config_user_file(config_file, recipe_name):
...
 
-    cfg['config_developer_file'] = _normalize_path(
-        cfg['config_developer_file'])
+    if cfg['quicklook']['active']:
+        cfg['config_developer_file'] = _normalize_path(
+            cfg['config_developer_file'])
+        cfg['quicklook']['output_dir'] = _normalize_path(
+            cfg['quicklook']['output_dir'])
+        cfg['quicklook']['recipe_dir'] = _normalize_path(
+            cfg['quicklook']['recipe_dir'])
```
Comment (schlunma) [inline esmvalcore/_config.py]: Can you add

```python
cfg['save_intermediary_cubes'] = False
cfg['remove_preproc_dir'] = False
```
?

I think these options should not be set in the quicklook mode.

Comment (bjoernbroetz) [reply esmvalcore/_config.py]: Done

Comment (schlunma) [reply esmvalcore/_config.py]: Ahh, sorry, I meant

```python
cfg['remove_preproc_dir'] = True
```

of course.

Comment (bjoernbroetz) [reply esmvalcore/_config.py]: Oh you test me :smile:

File: esmvalcore/_config.py [hunk 2]
```diff
@@ -60,9 +61,16 @@ def read_config_user_file(config_file, recipe_name):
...
     cfg['output_dir'] = _normalize_path(cfg['output_dir'])
     cfg['auxiliary_data_dir'] = _normalize_path(cfg['auxiliary_data_dir'])
-
     cfg['config_developer_file'] = _normalize_path(
         cfg['config_developer_file'])
+ 
+    if cfg['quicklook']['active']:
+       cfg['quicklook']['output_dir'] = _normalize_path(
+            cfg['quicklook']['output_dir'])
+        cfg['quicklook']['recipe_dir'] = _normalize_path(
```
Comment (schlunma) [inline esmvalcore/_config.py]: The test found an unexpected indent here

**Pull request merged**